### PR TITLE
l10n: uk: add initial translation

### PR DESCRIPTION
**Pull request by arkid15r**
This is a draft PR of resulting changes that I'm going to file later on https://github.com/git-l10n.

Please review locally as the PO file is too large.

### CODE REVIEW

File: po/uk.po
Comment (kasya) [inline po/uk.po:1]: There are some commented out lines in the end of the file, that need to be removed.

**Review by kasya — COMMENTED** [review]
👍🏼

File: po/uk.po
Comment (arkid15r) [reply po/uk.po:1]: 👍 

Done

**Review by kasya — COMMENTED** [review]
LGTM👍🏼

**Pull request closed without merging**